script.js

```diff
@@ -1,121 +1,138 @@
+  ctx.arc(0, 0, radius, 0, 2 * Math.PI);
+  ctx.fillStyle = "white";
+  ctx.fill();
+  
+  // Create's clock's outer grey border
+  var grad = ctx.createRadialGradient(0, 0, radius * borderInnerRadiusRatio, 0, 0, radius * borderOuterRadiusRatio);
+  grad.addColorStop(0, '#333');
+  grad.addColorStop(0.5, 'white');
+  grad.addColorStop(1, '#333');
+  ctx.strokeStyle = grad;
+  ctx.lineWidth = radius * borderWidthRatio;
+  ctx.stroke();
+  
+  //Create's inner circle 
+  ctx.beginPath();
+  ctx.arc(0, 0, radius * innerCircleRatio, 0 , 2 * Math.PI);
+  ctx.fillStyle = '#333';
+  ctx.fill();
+}
 
-    grad = ctx.createRadialGradient(0, 0, radius * 0.95, 0, 0, radius * 1.05);
-    grad.addColorStop(0, '#333');
-    grad.addColorStop(0.5, 'white');
-    grad.addColorStop(1, '#333');
-    ctx.strokeStyle = grad;
-    ctx.lineWidth = radius * 0.1;
-    ctx.stroke();
-    
-    ctx.beginPath();
-    ctx.arc(0, 0, radius * 0.1, 0 , 2 * Math.PI);
-    ctx.fillStyle = '#333';
-    ctx.fill();
+// Drawing the clock's numbers all horizontal
+function drawNumbers(ctx, radius) {
+  ctx.font = radius * fontSizeRatio + "px arial";
+  ctx.textBaseline = "middle";
+  ctx.textAlign = "center";
+  for(var num = 1; num <= NUM_ON_CLOCK; num++){
+    var ang = num * Math.PI / 6;
+    ctx.rotate(ang);
+    ctx.translate(0, -radius * numDistanceRatio);
+    ctx.rotate(-ang);
+    ctx.fillText(num.toString(), 0, 0);
+    ctx.rotate(ang);
+    ctx.translate(0, radius * numDistanceRatio);
+    ctx.rotate(-ang);
+  } 
 }
 
-// Drawing the clock's numbers
+// Get's the time and then draws hands 
+function drawTime(ctx, radius){
+  var now = new Date();
+  var hour = now.getHours();
+  var minute = now.getMinutes();
+  var second = now.getSeconds();
 
-function drawClock() {
-    drawFace(ctx, radius);
-    drawNumbers(ctx, radius);
-    drawTime(ctx, radius);
-  }
-  function drawNumbers(ctx, radius) {
-    var ang;
-    var num;
-    ctx.font = radius * 0.15 + "px arial";
-    ctx.textBaseline = "middle";
-    ctx.textAlign = "center";
-    for(num = 1; num < 13; num++){
-      ang = num * Math.PI / 6;
-      ctx.rotate(ang);
-      ctx.translate(0, -radius * 0.85);
-      ctx.rotate(-ang);
-      ctx.fillText(num.toString(), 0, 0);
-      ctx.rotate(ang);
-      ctx.translate(0, radius * 0.85);
-      ctx.rotate(-ang);
-    }
-  }
-//   Get's the time and then draws it on the canvas
-  function drawTime(ctx, radius){
-    var now = new Date();
-    var hour = now.getHours();
-    var minute = now.getMinutes();
-    var second = now.getSeconds();
-    //hour
-    hour = hour%12;
-    hour = (hour*Math.PI/6)+(minute*Math.PI/(6*60))+(second*Math.PI/(360*60));
-    drawHand(ctx, hour, radius*0.5, radius*0.07);
-    //minute
-    minute = (minute*Math.PI/30)+(second*Math.PI/(30*60));
-    drawHand(ctx, minute, radius*0.8, radius*0.07);
-    // second
-    second = (second*Math.PI/30);
-    drawHand(ctx, second, radius*0.9, radius*0.02);
-  }
+  hour = hour%12;

```

Comment:
space between hour % 12, also you should probably just declare new variables instead of modifying it in place